cmd/release-controller/http.go

```diff
@@ -280,10 +280,12 @@ func (c *Controller) apiReleaseLatest(w http.ResponseWriter, req *http.Request)
 	}
 
 	downloadURL, _ := c.urlForArtifacts(latest.Name)
+	digest, _ := c.digestForArtifacts(latest.Name)
```

Comment:
This is obviously not implemented.

Reply:
The `locateLatest` function returns a `TagReference`. That object has a `From` field which holds information like the `Name` and `Namespace` of the `ImageStreamTag`, `ImageStreamImage` or `DockerImage` that it is referencing. If you use the client to fetch the requisite item, the digest is:

 - `ImageStreamTag.Image.ObjectMeta.Name`
 - `ImageStreamImage.Image.ObjectMeta.Name`

In the case of a `DockerImage` you would need to parse out the `Name` field from the reference itself. I am not sure we need to handle this case.